.github/workflows/dhis2-artifacts.yml

```
@@ -8,7 +8,7 @@ env:
 jobs:
     publish:
         runs-on: ubuntu-latest
-        if: "!contains(github.event.head_commit.message, '[skip ci]')"
+        if: "!contains(github.event.head_commit.message, '[skip ci]') && !github.event.push.repository.fork && github.actor != 'dependabot[bot]'"
```

```suggestion
        if: "!github.event.push.repository.fork && github.actor != 'dependabot[bot]'"
```

This should be enough, GH nowadays respects `[skip ci]` natively.